case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+---
+title: Azure DevOps Case Study
+keywords: development platforms, Azure DevOps, Microsoft
+tags: Development Platforms, Azure DevOps, Microsoft
+sidebar: development-platforms
+permalink: development-platforms/case-study-azure-devops.html
+folder: development-platforms
+---
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [Brief History](#brief-history)
+3. [Features](#features)
+4. [Strengths](#strengths)
+5. [Weaknesses](#weaknesses)
+6. [Pricing](#pricing)
+7. [Comparison](#comparison)
+8. [How To](#how-to)
+9. [Summary](#summary)
+10. [Credits](#credits)
+11. [References](#references)
+
+## **Introduction**
+
+<div style="width: 600px;">
+
+![Azure DevOps offerings](https://miro.medium.com/max/800/1*8orwInnxqPRhrcKf9aOo9Q.png)
+
+<p style="padding: 0; margin: 0;">
+
+*Source: google*
+
+</p>
+
+</div>
+
+**Azure DevOps** was initially released in 2018 by Microsoft. It is a set of developer tools, services, and features that enable teams to plan, develop, test, and deliver software more effectively. Azure DevOps integrates with the broader Azure cloud computing platform, enabling users to take advantage of its features and services, such as continuous integration and deployment, and virtual machines. Azure DevOps has since become one of the leading development platforms and is widely used by development teams across the world.
+
+Azure DevOps is for the most part free to use, however there is some limitations with using only a free tier option. I will go in deeper details about the pricing [here](#pricing)
```

Comment:
```suggestion
Azure DevOps is for the most part free to use, however there is some limitations with using only a free tier option. We will look in deeper detail about the pricing [here](#pricing)
```

Reply:
In technical literature there is no `I`, only `we`, `you`, and `the author`.

Reply:
My preferred style (personally, this isn't a rule) is to use `we` to make the reader feel like a part of the process.